overrides/main.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}
+
+{% block announce %}
+    <b>&#x1F680 The 2025 ACCESS Workshop & Training Day is on its way! &#x1F4DA &#x2728 Everything you need for the ACCESS Training Day <a href="https://docs.access-hive.org.au/community_resources/access_workshop_2025/">will live right here!</a></b>
```

Comment:
```suggestion
    <b>&#x1F680 <a href="https://docs.access-hive.org.au/community_resources/access_workshop_2025/">ACCESS Workshop & Training Day 2025</a> &#x1F4DA</b>
```

I often like to link information directly in the content (i.e., without saying "it's here", or "in this page", but rather by linking directly the relevant portion, in this case "ACCESS Workshop & Training Day"). 
I think it's more direct and clear.

I would also:
- Change the background color of the banner, so it has contrast and it's more visible immediately by users.
- Centralise the content of the banner
- Depending on the banner background color, we could also change the linked text color

Reply:
Thanks for this, @KAUR1984! I agree with @atteggiani about concise wording and making more visible - I hardly noticed it when I clicked on the above Preview link...